src/ARToolkit.Maui/Handlers/ARSceneViewHandler.cs

```diff
@@ -155,8 +182,21 @@ public static void MapRenderPlanes(ARSceneViewHandler handler, IARSceneView arSc
         public static void MapRenderVideoFeed(ARSceneViewHandler handler, IARSceneView arSceneView)
         {
 #if !NETSTANDARD
-            if (handler.PlatformView != null)
-                handler.PlatformView.RenderVideoFeed = arSceneView.RenderVideoFeed;
+            if (handler.PlatformView is ARToolkit.ARSceneView arview)
+            {
+                arview.RenderVideoFeed = arSceneView.RenderVideoFeed;
+                if (arSceneView.RenderVideoFeed)
+                {
+                    arSceneView.SpaceEffect = arview.SpaceEffect = SpaceEffect.None;
+                    arSceneView.AtmosphereEffect = arview.AtmosphereEffect = AtmosphereEffect.None;
+                }
+                else
+                {
+                    arSceneView.SpaceEffect = arview.SpaceEffect = SpaceEffect.Stars;
+                    arSceneView.AtmosphereEffect = arview.AtmosphereEffect = AtmosphereEffect.HorizonOnly;
+                }
```

Comment:
Shouldn't the native control do this? If it's to sync up with native control, can we get a property change to react to, or just grab the value from `arview` instead of assigning both?